src/test/java/org/prgrms/prgrmsspring/repository/user/DBCustomerRepositoryTest.java

```diff
@@ -0,0 +1,222 @@
+package org.prgrms.prgrmsspring.repository.user;
+
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.prgrms.prgrmsspring.entity.user.Customer;
+import org.prgrms.prgrmsspring.exception.DataAccessException;
+import org.prgrms.prgrmsspring.exception.ExceptionMessage;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.util.List;
+import java.util.Optional;
+import java.util.UUID;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+@Transactional
+@SpringBootTest
```

Comment:
```suggestion
@JdbcTest
@ContextConfiguration(classes = {DBCustomerRepository.class})
@AutoConfigureTestDatabase(replace = AutoConfigureTestDatabase.Replace.NONE)
```

Repository 관련된 Context만 불러오는 것에 대해서 찾아보셨나요?
위 어노테이션에 대해서 한번 찾아보시죠!

Reply:
네 해당 내용은 제가 따로 개인 블로그에 공부해서 정리하겠습니다